src/vsr/replica.zig

```diff
@@ -7384,16 +7392,21 @@ pub fn ReplicaType(
             assert(self.commit_stage == .idle);
```

Comment:
Can `transition_to_normal_*` and `transition_to_view_change` assert that `self.log_view >= self.superblock.working.vsr_state.checkpoint.header.view`?